shell/browser/api/electron_api_protocol.cc

```diff
@@ -89,6 +89,16 @@ std::vector<std::string> GetStandardSchemes() {
   return g_standard_schemes;
 }
 
+void AddServiceWorkerScheme(const std::string& scheme) {
```

Comment:
Can you move this helper function under `shell/common`, `shell/renderer` should not interact with `shell/browser/` directly.

Also can you add a comment of how this call in the renderer is different from the one we already do in the renderer https://github.com/electron/electron/blob/d74ad51826c0672b72e807de580cb97c3b29a8bb/shell/renderer/renderer_client_base.cc#L223 , looks like the latter is used for checks in the registration stage while the former is used in checks that determine the creation of url loader that can intercept requests in the worker.

Reply:
@deepak1556, this PR has already been merged. How would you like me to proceed with those changes?

Reply:
You can create a new PR as a follow-up. Thanks!